<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMasterDataFormClass</class>
 <widget class="QDialog" name="SearchMasterDataFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>マスター検索画面</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QTreeWidget" name="trMasterList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="leMasterID">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>44</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="PasswordPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
    </property>
   </widget>
   <widget class="PasswordPushButton" name="pbReadMasterData">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Master ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Master Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMasterRemark">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>164</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMasterName">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>104</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Remark</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMasterImage">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Master ID</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSearchByMasterID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>15</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="PasswordPushButton" name="pbSearchByMasterID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>12</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="PasswordPushButton" name="pbSearchByMasterName">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>77</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Master Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSearchByMasterName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>14</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="twMasterList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>861</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PasswordPushButton</class>
   <extends>QPushButton</extends>
   <header>passwordpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
